README.md

```diff
@@ -2,11 +2,11 @@
 # Cybereason
 
 Publisher: Cybereason  
-Connector Version: 2\.3\.0  
+Connector Version: 2\.3\.1  
 Product Vendor: Cybereason  
 Product Name: Cybereason  
 Product Version Supported (regex): "\.\*"  
-Minimum Product Version: 5\.3\.0  
+Minimum Product Version: 5\.3\.5  
```

Comment:
why the min version is 5.3.5?

Reply:
This was updated by the cybereason team. But they forgot to update in the Readme

Reply:
ok